src/components/molecules/NavbarSchedule/NavbarSchedule.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback, useState } from "react";
+import classNames from "classnames";
+
+import { useSovereignVenueId } from "hooks/useSovereignVenueId";
+
+import { SchedulePageModal } from "components/organisms/SchedulePageModal/SchedulePageModal";
+
+import "./NavbarSchedule.scss";
+
+export const NavbarSchedule: React.FC = () => {
+  const { sovereignVenueId: navbarTitle } = useSovereignVenueId();
```

Comment:
This is getting the `venueId`, not the `venueName`. I believe we should probably be using `venueName` here shouldn't we?